rj_gameplay/classifier/fumble_classifier.py

```diff
@@ -0,0 +1,171 @@
+import stp.rc as rc
+import rj_gameplay.gameplay_node as gameplay_node
+import rclpy
+import math.atan2 as atan2
+from typing import Union
+
+
+# def fumble_classifier(world_state: rc.WorldState, possess) -> bool:
+# 	if possess is None:
+# 		return True
+# 	else: 
+# 		return False
+
+"""
+get_distance returns the distance between the ball and the player
+robot_id: robot's id
+"""
+def get_distance(world_state: rc.WorldState, robot_id) -> float:
+	if world_state is not None:
+		robot = world_state.our_robots[robot_id]
+		pose_x = robot.pose[0]
+		pose_y = robot.pose[1]
+		ball_pos_x = world_state.ball.pos[0]
+		ball_pos_y = world_state.ball.pos[1]
+		distance = ((ball_pos_x-pose_x)**2 + (ball_pos_y-pose_y)**2)**0.5
+		return distance
+		
+
+#get_recipient guessing the intended recipient
+def get_recipient(world_state: rc.WorldState, team, vel_angle):
+	ball_pos_x = world_state.ball.pos[0]
+	ball_pos_y = world_state.ball.pos[1]
+	angle = 6.28
+	if team == 'our':
```

Comment:
probably just change team to be a boolean instead of a string. So instead of team it could be an our_team variable that is either True or False.